### PR TITLE
OR: finishing Laravel

### PR DESCRIPTION
**Pull request by emarcey**

### CODE REVIEW

File: reports/Oregon.md [hunk 1]
```diff
@@ -82,6 +82,45 @@ id: or-portland-5
...
+### Tear gas fired across active street at protestors | (Believed to be) June 2nd
+
+In events occurring either late on June 2nd or in the early morning of June 3rd, police can be heard declaring unlawful assembly and seen firing tear gas across the street at gathered protestors. Some protestors appear to throw tear gas canisters back.
+
+There are numerous passing cars that appear unrelated to the protest but are caught in the gas. They struggle to maneuver around canisters & clouds of tear gas.
+
+tags: tear-gas
+
+id: or-portland-22
+
```
Comment (jcapobianco-cbi) [inline reports/Oregon.md:97]: missing link here. did something get confused with or-portland-21?

Comment (jcapobianco-cbi) [reply reports/Oregon.md:97]: or did the link somehow get moved up in the first line of this diff section?

File: reports/Oregon.md [hunk 2]
```diff
@@ -82,6 +82,45 @@ id: or-portland-5
...
+### Police SUV accelerates towards protestors | June 3rd
+
+In helicopter footage from KGW, protestors are seen moving police barricades and traffic cones on the street. It is unclear if protestors are attempting to clear the street or form a barricade. 
+
+A police SUV accelerates into the intersection, swerving towards protestors, which forces them to run out of the way. 
+
+tags: vehicle
+
+id: or-portland-23
```
Comment (jcapobianco-cbi) [inline reports/Oregon.md:123]: no links

File: reports/Oregon.md (hunk 1 above)
Comment (emarcey) [reply reports/Oregon.md:97]: Fuck me this is going to be a nightmare to figure out

**Pull request merged**